ngx-tools/src/jwt-token-managment/effects.ts

```diff
@@ -57,7 +61,7 @@ type FullClaimsTuple = [
 export class JwtTokenProviderEffects {
 
   @Effect()
-  initializationCleanup$ = this.actions$
+  initializationCleanup$ = of(true)
```

Comment:
Decided this was a better way to issue an action on start vs listening to anything that happens